examples/exampleproject/modules/optionals/buckets/deployspec.yaml

```diff
@@ -9,10 +9,8 @@ deploy:
       commands:
       # execute the CDK
       - cdk deploy --require-approval never --progress events --app "python app.py" --outputs-file ./cdk-exports.json
-      # derive the metadata environment parameter name that seedfarmer is looking for from the CODESEEDER project name
-      - export P_METADATA=$(echo $AWS_CODESEEDER_NAME | tr '[:lower:]' '[:upper:]')_MODULE_METADATA
-      # assign that paramter name the output from CDK by calling a custom python script
-      - export ${P_METADATA}=$(python export_metadata.py)
+      # get the cdk.output and write to a file named <project>_MODULE_METADATA --- ALL UPPERCASE AND '_'
+      - python export_metadata.py
```

Comment:
Just curious: Do I need to change from `export <project-name>_MODULE_METADATA` to now writing such that file or is both supported? (I prefer the latter).

Reply:
you do not need to change...you are free to use whatever means (elegant or ugly) to get the data you want persisted to SSM in json format.

Reply:
We are thinking of removing the need for the `project` prefix to make the parameter names more easily transferable.